cmd/api/src/model/sso_provider.go

```diff
@@ -27,7 +27,7 @@ const (
 
 // SSOProvider is the common representation of an SSO provider that can be used to display high level information about that provider
 type SSOProvider struct {
-	Type SSOProviderType `json:"type"`
+	Type SessionAuthProvider `json:"type"`
```

Comment:
This likely needs to be updated to whatever was reflected in https://github.com/SpecterOps/BloodHound/pull/896